SpatialGDK/Documentation/content/tutorials/multiserver-shooter/tutorial-multiserver-cloudtest.md

```diff
@@ -7,6 +7,10 @@
 
 An assembly is what’s created when you run `BuildWorker.bat`. They’re .zip files that contain all the files that your game uses when running in the cloud.
 
+**Note:** You must close the Unreal Editor before building your workers. If the Editor is open when you try to build your workers the command will fail.
+
+**Note:** By default, Spatial networking is enabled for Unreal GDK projects. However, if the `bSpatialNetworking` option is visible in your `DefaultGame.ini` configuration file (located in `<ProjectRoot>\Game\Config` directory), ensure that it is set to `True` to enable Spatial networking for your cloud deployment.
```

Comment:
New unexplained stuff in a tutorial or Get Started guide alert!
(Same comments as above.)